MedicalDocManagment.WebUI/app/users-list/page/item.component.ts

```diff
@@ -0,0 +1,24 @@
+﻿import { Component, Input } from '@angular/core';
+import { Response } from '@angular/http';
+
+import { ItemActionListComponent } from './item-actionlist.component';
+import UserModel from '../../models/usermodel'
+
+@Component({
+  moduleId: module.id,
+  selector: '[item]',
+  templateUrl: './views/item.component.html',
+  styleUrls: ['./views/item.component.css'],
+})
+
+/**
+ * Class, which implements item from list of users.
+ */
+export class ItemComponent {
```

Comment:
Goal of comment: **Please, change name of the class on more suitable.**
Description: I mean that if this class is `...implements item from list of users`, do the name more understandable. 
You could rename it on `UsersListItemComponent`. 
And do same work with the selector of this class.
Because this class is avaliable to export to another component and somebody can use the name `ItemComponent`. Also it's too abstractly.
PS: Мені б мемуари писати ;-)